qt/scientific_interfaces/EnggDiffraction/EnggDiffractionPresenter.cpp

```diff
@@ -784,13 +779,13 @@ void EnggDiffractionPresenter::inputChecksBeforeCalibrate(
     throw std::invalid_argument("The Ceria number" + g_runNumberErrorStr);
```

Comment:
This refers to the run number for the calibration sample, which is always a ceria sample

Reply:
Would this also be true for IMAT? Are we sure `calib_sample_num` or something isn't clearer?

Reply:
a quick look at JournalView seems like IMAT also have a ceria calibration sample. ENGINX use 2 calibration samples for each reduction - a vanadium and a ceria, so i think it would potentially be confusing to name just one of these 'calibration_sample'. if we come up with a use case where a different material is needed for the calibration, there might be a bit of 'crossing that bridge when we come to it'

Reply:
Ok. I'm happy with this then.